README.md

```diff
@@ -141,29 +143,28 @@ In order to be able to test your localhost from a VM (such as VirtualBox) you'll
 
 Valtech follows RPS's `rebase` strategy - working off a `DW-0000-master` branch, which is kept in line with `master` daily, and all feature branches are created off `DW-0000-master`. Once a feature branch is complete, we squash all commits on it into 1 meaningful commit and merge it into `DW-0000-master` - 1 at a time.
 
-### Step 0 - Checkout `DW-0000-master` branch
+### Step 0 - Checkout `master` branch
 ```
-git checkout DW-0000-master
+git checkout master
 ```
 
 ### Step 1 - Getting `DW-0000-master` up to date with `master` before creating a new feature branch
 ```
-git pull --rebase origin DW-0000-master // pull latest updates of the branch from origin
 git pull --rebase origin master // to make sure it's up to date locally
```

Comment:
you don't need to, and never should, rebase master branch.
all you need is:

```
git fetch
git create-branch DW-XX-my-feature
work, work, work
git fetch
git rebase -i origin/master
```

all already described in here https://github.com/NHS-digital-website/hippo/blob/master/CONTRIBUTING.md

the "create-branch" is an alias that you get by running `make init`. It's shortcut for `git branch` and `git checkout`

Reply:
Thanks @wojtek-oledzki for the heads up, this is an outdated piece of documentation, which somehow made it through to the README. I'll update it to reflect our current rebasing strategy.